src/app/shared/expiration/date-picker/expiration-date.component.ts

```diff
@@ -78,15 +78,12 @@ export class ExpirationDateComponent implements OnInit, OnDestroy {
   }
 
   restoreDefaultHandler(): void {
+    this.defaultExpirationDate();
     this.add.emit(this.defaultValidityPeriod);
-    this.expirationDate.setValue(
-      new Date(Date.now() + this.defaultValidityPeriod)
-    );
-    this.expirationTimeShift = this.defaultValidityPeriod;
     this.hideRemoveButton = false;
   }
 
-  defaultExpirationDate() {
+  defaultExpirationDate(): void {
```

Comment:
I'm curious what your personal convention is for using the `set` prefix? Why (or why not) should this have `set` prefix (i.e. be `setDefaultExpirationDate()`). It seems as though `this.expirationDate` is being set so maybe it makes sense?

Reply:
added prefix, you're right. I'm setting there a value, so it's better to add `set`.